observer/monitor.go

```diff
@@ -0,0 +1,59 @@
+package observer
+
+import (
+	"time"
+
+	blog "github.com/letsencrypt/boulder/log"
+	p "github.com/letsencrypt/boulder/observer/probes"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// monitor contains the parsed, normalized, and validated configuration
+// describing a given oberver monitor
+type monitor struct {
+	name   string
+	period time.Duration
+	probe  string
+	prober p.Prober
+	logger blog.Logger
+	metric prometheus.Registerer
+}
+
+// start creates a ticker channel then spins off a prober goroutine for
+// each period specified in the monitor config and a timeout inferred
+// from that period. This is not perfect, it means that the effective
+// deadline for a prober goroutine will be TTL + time-to-schedule, but
+// it's close enough for our purposes
+func (m monitor) start() *time.Ticker {
+	ticker := time.NewTicker(m.period)
+	logInfo := func(status string, took string) {
+		m.logger.Infof(
+			"status=[%s] probe=[%s] duration=[%s] monitor=[%s]",
+			status, m.probe, took, m.name)
+	}
+	go func() {
+		for {
+			select {
+			case <-ticker.C:
+				success, took := m.prober.Do(m.period)
+				statTotalObservations.WithLabelValues(m.probe, m.name).Add(1)
+				if !success {
+					statTotalErrors.WithLabelValues(m.probe, m.name).Add(1)
+					logInfo("failure", took.String())
+				}
+				logInfo("success", took.String())
+			}
+		}
+	}()
+	return ticker
+}
+
+func (m monitor) New(c MonConf, log blog.Logger, prom prometheus.Registerer) *monitor {
```

Comment:
I don't think this method needs a receiver; it can just be `func New(c MonConf, ...) *monitor { ... }`. That lets you remove the predeclaration at observer.go:68 as well. And then move this constructor above `.start()`, too.